### ISSUE TITLE
Clearer API documentation?

### ISSUE THREAD

**Issue by genyded** [kadirahq/react-mounter#15]
Syntax differences without concise API documentation (what the actual API is, not sample code using a portion of it) make it hard to figure out what is allowed. For example, the readme has this example:

```
mount(MainLayout, {
  content: () => (<WelcomeComponent name="Arunoda" />)
});
```

And the [sample repo](https://github.com/mantrajs/meteor-mantra-kickstarter) has this:

```
mount(AuthCheckCtx, {
   LayoutDefault, content: () => (<Simplest title="Template..."/>)
});
```

...one has the layout component inside the object with another auth component outside and the other just outside. So can the object have nested components or just one and either way what args can and can't be inside and/or the object optionally or required?

Thanks

**Comment by zackng** [comment]
Agree. Can the documentation also show example pure js React component in use. 

Great job btw!

**Comment by sascha1337** [comment]
please some examples, need this 4 production THX
